src/gt4py/next/program_processors/runners/dace_fieldview/transformations/loop_blocking.py

```diff
@@ -568,6 +573,7 @@ def _rewire_map_scope(
                     piping_edge.data = dace.Memlet.from_array(
                         data_name, sdfg.arrays[data_name], piping_edge.data.wcr
                     )
+                    piping_edge.data.try_initialize(sdfg, state, piping_edge)
```

Comment:
Normally you create a full edge (src, dst and memlet) then DaCe will call this function for you.
Here we exchange the Memlet of an edge with a different one.
This is way we have to call the function explicitly.

This part of DaCe is in my view strange and not well designed, partly because the Memlet itself is not the connection between two nodes in the data flow graph, but just the payload of such an edge.